src/btree/bt_walk.c

```diff
@@ -163,12 +163,12 @@ __page_ascend(WT_SESSION_IMPL *session,
 }
 
 /*
- * __page_descend --
- *	Descend the tree one level.
+ * __page_descend_prev --
+ *	Descend the tree one level, during a previous-cursor walk.
  */
 static void
-__page_descend(WT_SESSION_IMPL *session,
-    WT_PAGE *page, WT_PAGE_INDEX **pindexp, uint32_t *slotp, bool prev)
+__page_descend_prev(
```

Comment:
For consistency this function should change name to `__ref_descend_prev` due to the parameter change.